### PR TITLE
fix(SQS): Log errors with SQS connections. Fixes #1114

### LINKED ISSUE #1114: SQS Event Source does not properly log connection errors
**Describe the bug**
When the SQS Event Source attempts to connect AWS SQS, it will return an error if there is an issue the connection. However, the error is never logged, because `StartListening` is run by [`common.Connect`](https://github.com/argoproj/argo-events/blob/master/eventsources/eventing.go#L344), the error is lost during retry steps, and the only error that appears in the logs is `failed to start listening eventsource`

**To Reproduce**
1. Create an SQS Event source 
    ```yaml
   apiVersion: argoproj.io/v1alpha1
   kind: EventSource
   metadata: 
     name: test-failing-sqs
   spec:
     sqs:
      sqs_message:
        jsonBody: true
        queue: does-not-exist
        region: us-east-1
    ```

2. Follow the logs of the event source pod

**Expected behavior**
An error log revealing the underlying issue.

**Environment (please complete the following information):**
 - Kubernetes: 1.16.15
 - Argo: 2.11.8
 - Argo Events: 1.1.0

---

**Message from the maintainers**:

If you wish to see this enhancement implemented please add a 👍 reaction to this issue! We often sort issues this way to know what to prioritize.

### PR DESCRIPTION
**Pull request by delwaterman**
Add additional logging to expose errors for AWS SQS
Fixes https://github.com/argoproj/argo-events/issues/1114

Checklist:

* [x] My organization is added to [USERS.md](https://github.com/argoproj/argo-events/blob/master/USERS.md).

### CODE REVIEW

File: eventsources/sources/awssqs/start.go [hunk 1]
```diff
@@ -70,6 +70,7 @@ func (el *EventListener) StartListening(ctx context.Context, dispatch func([]byt
 	var awsSession *session.Session
 	awsSession, err := awscommon.CreateAWSSessionWithCredsInVolume(sqsEventSource.Region, sqsEventSource.RoleARN, sqsEventSource.AccessKey, sqsEventSource.SecretKey)
 	if err != nil {
+		log.Error("Error creating AWS credentials", zap.Error(err))
```
[the remark this answers is not in the document]
Comment (delwaterman) [reply eventsources/sources/awssqs/start.go]: Fixed

File: eventsources/sources/awssqs/start.go [hunk 2]
```diff
@@ -85,6 +86,7 @@ func (el *EventListener) StartListening(ctx context.Context, dispatch func([]byt
 
 	queueURL, err := sqsClient.GetQueueUrl(getQueueURLInput)
 	if err != nil {
+		log.Error("Error getting SQS Queue URL", zap.Error(err))
```
[the remark this answers is not in the document]
Comment (delwaterman) [reply eventsources/sources/awssqs/start.go]: Fixed

**Comment by whynowy** [comment]
@delwaterman - we are experiencing some issues about Github actions, and that messed up the whole operations. Would you please merge the master again? that usually can be done in the UI, however it's not available now.

**Comment by delwaterman** [comment]
@whynowy Merged branch master as requested

**Review by whynowy — APPROVED** [review]
LGTM, thanks!

**Pull request merged**